.coderabbit/rules/react-key-prop.yml

```
@@ -0,0 +1,25 @@
+language: typescript
+message: "React list item missing 'key' prop. Each child in a list should have a unique 'key' prop."
+rule:
+  any:
+    - pattern: |
+        $ARRAY.map(($ITEM) => (
+          <$COMPONENT $$$PROPS />
+        ))
+      not:
+        has:
+          pattern: key=$KEY
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)